corehq/apps/data_interfaces/models.py

```diff
@@ -213,12 +252,21 @@ class AutomaticUpdateAction(models.Model):
         (ACTION_CLOSE, ACTION_CLOSE),
     )
 
+
     rule = models.ForeignKey('AutomaticUpdateRule', on_delete=models.PROTECT)
     action = models.CharField(max_length=10, choices=ACTION_CHOICES)
 
     # property_name and property_value are ignored unless action is UPDATE
     property_name = models.CharField(max_length=126, null=True)
     property_value = models.CharField(max_length=126, null=True)
 
+    property_value_type = models.CharField(max_length=15,
+                                           choices=PropertyTypeChoices.choices,
+                                           default=PropertyTypeChoices.EXACT)
+
     class Meta:
         app_label = "data_interfaces"
+
+
+# import signals
+from corehq.apps.data_interfaces import signals
```

Comment:
I believe the preferred approach is to import in `AppConfig.ready`
https://docs.djangoproject.com/en/1.10/ref/applications/#django.apps.AppConfig.ready